README.md

```diff
@@ -180,6 +180,9 @@ Set the rating for the current track
 
 - rating `String` - Rating to set for the current track. This should be between 1 and 5
 
+#### `rating.resetRating()`
+Resets the rating for the current track to the "neutral" rating
```

Comment:
I don't think "neutral" rating is the best way to phrase it. Maybe something like:

```
Removes existing rating from the current track
```